.github/workflows/rust.yml

```diff
@@ -26,11 +26,40 @@ jobs:
             pkg install -y curl
             curl https://sh.rustup.rs -sSf --output rustup.sh
             sh rustup.sh -y --profile minimal --default-toolchain stable
-            echo "~~~~ rustc --version ~~~~"
-            $HOME/.cargo/bin/rustc --version
-            echo "~~~~ freebsd-version ~~~~"
+          run: |
```

Comment:
Actually, could we do this with a matrix instead?

Also have the changes to the contents of the (FreeBSD) job in a separate commit (and squash all the other changes).